tests/utils.go

```diff
@@ -120,7 +120,13 @@ const (
 	NamespaceTestAlternative = "kubevirt-test-alternative"
 )
 
-const LocalStorageClass = "local"
+const (
+	LocalStorageClass = "local"
+
+	HostPathStorageClass = "host-path"
+
+	BlockVolumeStorageClass = "block-volume"
```

Comment:
should this be `local-block`?

Reply:
Good question!  Not sure it matters.  But I think it may actually be better to use a different storage class.  

The PVC in local-block-storage is referenced by one of our example manifests (vmi-block-pvc.yaml) and is not used in any functional tests.  The PVs created in the functional tests share the same block device as vmi-block-pvc.yaml, though.  And I think if you are going to have two PVs share the same device it would be better to have them in different storage classes.  Although it's probably still a bad situation.  Perhaps I should allocate another block device for functional tests?

Reply:
Hi, the storage class has to be "local" for the block volume PVs, because they are handled by the local storage provider, which is responsible for the "local" storage class, and has access to the actual block device which is used (see https://github.com/kubevirt/kubevirt/blob/master/images/disks-images-provider/entrypoint.sh#L25)

Reply:
and this is the mentioned local storage provisioner: https://github.com/kubevirt/kubevirtci/blob/master/manifests/local-volume.yaml

Reply:
@slintes the local volume provisioner (called "local") that is created by kubevirtci does not do anything with block volumes.  Look at the code here:  https://github.com/kubernetes-incubator/external-storage/tree/master/local-volume/provisioner.  It creates PVs from directories on a filesystem and only deals with the PVs it creates.
 
And having block volumes share the same storage class name was causing errors with upload tests because a block volume was being assigned to the upload server pod instead of a filesystem PV.

Let's look at the results of the tests (if they ever complete) to see if changing the storage class name for block volumes causes a test failure.  I don't think it does.

Reply:
@slintes whoops!  it does handle block PVs.  I apologize.  But in my testing it appears that having a storage class that has both block and filesystem PVs, weirdness occurs.  So maybe we should think about about having separate provisioners.

Reply:
I'm not familiar yet with the upload, but I assume you create a PVC or a PV? Did you try setting volumeMode=Filesystem on it?

Reply:
mh, but probably you don't want to care about the mode, and it's only an issue in this special environment. So, maybe using 2 provisioners is the way to go indeed.